nemo/collections/tts/torch/tts_tokenizers.py

```diff
@@ -201,6 +201,7 @@ def __init__(
         pad_with_space=False,
         non_default_punct_list=None,
         text_preprocessing_func=german_text_preprocessing,
+        phonemes=None,
```

Comment:
`phonemes` is a binary variable. So better to set it default `False` instead of `None`.